python-app/goss/action.yml

```diff
@@ -0,0 +1,57 @@
+name: Goss
+description: Test image with Goss
+
+inputs:
+  GOSS_IMAGE:
+    description: "name used for goss temporary image"
+    required: true
+  GOSS_FILES_PATH:
+    description: "path to goss file"
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: check for goss file
+      id: goss_file
+      run: |
+        if [[ -d ${{ inputs.GOSS_FILES_PATH }} ]]; then
+          echo goss_enabled=true >> $GITHUB_ENV
+        else
+          echo goss_enabled=false >> $GITHUB_ENV
+        fi
+      shell: bash
```

Comment:
Actually, can this be done before `checkout`? It is looking for a checked-in file.